addons/l10n_in/views/report_invoice.xml

```diff
@@ -30,6 +22,28 @@
             </td>
         </xpath>
 
+        <xpath expr="//div[@id='qrcode_info']" position="attributes">
+            <attribute name="t-if" add="and o.company_id.account_fiscal_country_id.code != 'IN'" separator=" "/>
+        </xpath>
+
+        <xpath expr="//div[@id='qrcode_info']" position="after">
+            <t t-if="o.company_id.account_fiscal_country_id.code == 'IN'">
+                <div style="display:-webkit-flex;" class="flex-column">
```

Comment:
Maybe stupid question but `d-flex` class from bootstrap doesn't make this stuff for you?

Reply:
The reason we need the term webkit is because it's the only way to make flexbox with the wkhtmltopdf library. Therefore just using Bootstrap will work on the preview of an invoice but not on pdf.